src/liger_kernel/transformers/jsd.py

```diff
@@ -4,8 +4,12 @@
 
 
 class LigerJSD(nn.Module):
-    def __init__(self):
+    def __init__(self, beta=0.5):
         super().__init__()
+        assert (
+            beta > 0 and beta < 1
+        ), f"beta must be greater than 0 and less than 1. Got: {beta}"
+        self.beta = beta
 
-    def forward(self, p, q):
-        return LigerJSDFunction.apply(p, q)
+    def forward(self, log_q, log_p):
```

Comment:
This is the correct order of input and target (student and teacher) respectively. would it be too confusing?

Reply:
yeah, the name is a bit confusing, or we can add some descriptions here to clarify